src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -480,4 +480,10 @@ Authorization rules consists of:
 - **authorized operations** (`operations`): which operations are allowed for the given strategy and provider. If not specified, `create`, `read`, `update`, and `delete` operations are allowed.
   - **`read` operation**: `read` operation can be replaced with `get`, `list`, `sync`, `listen`, and `search` for a more granular query access
 
+<Callout warning>
+
+  It is **not** recommended to use granular operations (`get`, `list`, `sync`, `listen`, `search`) instead of `read` if you use the Datastore feature of Amplify libraries, in which case `listen` and `sync` are **required** for its functionality.
```

Comment:
```suggestion
If you use DataStore instead of the API category to connect to your AppSync API, then you must allow `listen` and `sync` operations for your data model.
```